src/app/bisoncoin/src/components/Button.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <button class="button">{{ label }}</button>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+
+@Component
+export default class Button extends Vue {
+  @Prop() private label!: string;
+}
+</script>
+
+<style>
+.button {
+  background-color: #00a3e050;
```

Comment:
Should store all colours and text in separate asset files and then reference them in components and styles